app/controllers/job/tasklist.js

```diff
@@ -15,6 +16,11 @@ export default class JobTasklistController extends Controller {
   page = 0;
   size = 15;
 
+  @action
+  reload() {
+    this.router.refresh();
```

Comment:
I think calling refresh without a pivot route is a footgun since it also refreshes the `application` route hooks (which might have "setup" logic that assumes it only runs once). For this app it doesn't matter since it doesn't have a custom application route class yet, but we might add it in the future so I would add a pivot route to be sure.